.pre-commit-config.yaml

```diff
@@ -20,6 +20,11 @@ repos:
     exclude_types:
       - "markdown"
   - id: check-case-conflict
+- repo: https://github.com/python-poetry/poetry
+  rev: 1.2.2
+  hooks:
+    - id: poetry-check
+    - id: poetry-lock
```

Comment:
Not a reviewer but — this is going to update the lock file on every run — is that what we want?